pkg/buses/message.go

```diff
@@ -27,6 +27,8 @@ var forwardHeaders = []string{
 }
 
 var forwardPrefixes = []string{
+	// knative
+	"x-knative-",
```

Comment:
maybe just `knative-` since the `x-` prefix is generally discouraged

Reply:
ok